src/components/NewCommentForm.tsx

```diff
@@ -35,37 +37,59 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
     });
   };
 
+  const addComment = async (item: CommentData) => {
+    if (!post) {
+      return;
+    }
+
+    try {
+      const newComment = await commentsApi.createComment({
+        name: item.name,
+        email: item.email,
+        body: item.body,
+        postId: post.id,
+      });
+
+      dispatch(commentsActions.addComment(newComment));
+    } catch (error) {
+      dispatch(commentsActions.changeError(true));
+    }
+  };
+
   const handleChange = (
     event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
   ) => {
     const { name: field, value } = event.target;
 
-    setValues(current => ({ ...current, [field]: value }));
-    setErrors(current => ({ ...current, [field]: false }));
+    setValues((current) => ({ ...current, [field]: value }));
+    setErrors((current) => ({ ...current, [field]: false }));
```

Comment:
You don't need to use curly brackets around your single argument if you have one-line syntax.